components/director/internal/model/eventapi.go

```diff
@@ -67,6 +68,23 @@ func (e *EventDefinitionInput) ToEventDefinition(id, appID, tenant string) *Even
 	}
 }
 
+func (e *EventDefinitionInput) ToEventDefinitionWithPackageID(id string, pkgID *string, tenant string) *EventDefinition {
```

Comment:
What do you think about a name `ToEventDefinitionWithinPackage`? And also please add TODO comment that eventually we will replace current `ToEventDefinition` method with this logic.